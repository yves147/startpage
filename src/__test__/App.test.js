import React from 'react';
import { render } from '@testing-library/react';
import { config } from '../__test__/config'
import App from '../App';

describe('Test hiccup', () => {
  test('should render page correctly', () => {
    const { container } = render(<App config={config} />)
    expect(container).toMatchSnapshot()
    
  })
  
})
