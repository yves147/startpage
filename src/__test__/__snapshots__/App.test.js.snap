// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test hiccup should render page correctly 1`] = `
<div>
  <div
    class="section featured"
  >
    <a
      href="http://google.com"
      rel="noopener noreferrer"
      target="_blank"
    >
      <li
        class="card container "
        style="background-image: url(./assets/logo.png);"
      >
        Featured Link
      </li>
    </a>
    <a
      href="http://google.com"
      rel="noopener noreferrer"
      target="_blank"
    >
      <li
        class="card container "
        style="background-image: url(./assets/card.png);"
      >
        Another Feaured link
      </li>
    </a>
  </div>
  <div
    class="section categories"
  >
    <div
      class="category"
    >
      <h1
        class="title"
      >
        Category 1
      </h1>
      <ul>
        <a
          href="http://google.com"
          rel="noopener noreferrer"
          target="_blank"
        >
          <li
            class="card container "
          >
            <span
              class="name"
            >
              Link 1
            </span>
            <span
              class="link"
            >
              http://google.com
            </span>
          </li>
        </a>
        <a
          href="http://google.com"
          rel="noopener noreferrer"
          target="_blank"
        >
          <li
            class="card container "
          >
            <span
              class="name"
            >
              Test Link 2
            </span>
            <span
              class="link"
            >
              http://google.com
            </span>
          </li>
        </a>
      </ul>
    </div>
    <div
      class="category"
    >
      <h1
        class="title"
      >
        Category 2
      </h1>
      <ul>
        <a
          href="http://google.com"
          rel="noopener noreferrer"
          target="_blank"
        >
          <li
            class="card container "
          >
            <span
              class="name"
            >
              Link 1
            </span>
            <span
              class="link"
            >
              http://google.com
            </span>
          </li>
        </a>
        <a
          href="http://google.com"
          rel="noopener noreferrer"
          target="_blank"
        >
          <li
            class="card container "
          >
            <span
              class="name"
            >
              Link 2
            </span>
            <span
              class="link"
            >
              http://google.com
            </span>
          </li>
        </a>
      </ul>
    </div>
  </div>
  <svg
    class="config-icon"
  >
    cog-solid.svg
  </svg>
</div>
`;
